Readme.md

```diff
@@ -21,7 +21,7 @@ var localize = require('node-yaml-localize');
 
 app.use(localize.middleware({
     path: __dirname + "/config/locales",
-    locale: function() {
+    locale: function(req, res) {
```

Comment:
need to aadd parameters for req and res, otherwise req is not defined inside the "locale" function